source/CommonJobs/CommonJobs.Application.EvalForm/EmployeeSearching/EvaluationToDelete_Search.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CommonJobs.Domain.Evaluations;
+using Raven.Client.Indexes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommonJobs.Application.EvalForm.EmployeeSearching
+{
+    class EvaluationToDelete_Search : AbstractMultiMapIndexCreationTask<EvaluationToDelete_Search.Projection>
+    {
+        public class Projection
```

Comment:
I'm trying to avoid the projection, but i think it's clearer to see the result structure.